core/src/main/java/org/geysermc/geyser/translator/collision/SpawnerCollision.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright (c) 2019-2022 GeyserMC. http://geysermc.org
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- *
- * @author GeyserMC
- * @link https://github.com/GeyserMC/Geyser
- */
-
-package org.geysermc.geyser.translator.collision;
-
-import lombok.EqualsAndHashCode;
-
-@EqualsAndHashCode(callSuper = true)
-@CollisionRemapper(regex = "^spawner$")
-public class SpawnerCollision extends SolidCollision {
-    public SpawnerCollision(String params) {
```

Comment:
Collisions are discovered at runtime. You can make IDEA suppress the warning if the class/method is annotated with a specific annotation. It should be one of the context actions for fixing the warning

Reply:
This particular spawner collision fix links a resolved Minecraft Bedrock issue on the issue tracker though, so I'd assume this isn't needed anymore